.github/workflows/specimin_evaluation_CI.yml

```diff
@@ -0,0 +1,204 @@
+          fetch-depth: 0
+
+      - name: Debug - Show Current Branch
+        run: |
+          set -ex
+          echo "Current branch: $(git branch --show-current)"
+
+      - name: Debug - List Files in Repository
+        run: |
+          set -ex
+          echo "Current directory: $(pwd)"
+          echo "Listing files in /home/runner/work/specimin/specimin"
+          ls -la /home/runner/work/specimin/specimin
+          
+          
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Set up Java JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '21'
+          distribution: 'adopt'
+          architecture: 'x64'
+          server-password: ${{ secrets.GITHUB_TOKEN }}
+          overwrite-settings: true
+          check-latest: false
+
+      - name: Install dependencies
+        run: |
+          set -ex  # Exit immediately if a command exits with a non-zero status and print commands as they are executed
+          python -m pip install --upgrade pip
+
+      - name: Display CSV File Contents loaded in working environment
+        run: |
+          set -ex
+          if [ -f /home/runner/work/specimin/specimin/CI_repository_list.csv ]; then
+            cat /home/runner/work/specimin/specimin/CI_repository_list.csv
+          else
+            echo "File /home/runner/work/specimin/specimin/CI_repository_list.csv does not exist"
+            exit 1
+          fi
+
+      - name: Clone ASHE Project 
+        run: |
+          set -ex
+          git clone https://github.com/njit-jerse/ASHE_Automated-Software-Hardening-for-Entrypoints ASHE
+          
+      - name: Create ASHE Clone SPACE Directory
+        run: |
+         set -ex
+         mkdir -p $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+         chmod 777 $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+         ls -ld $(pwd)/ASHE/CI_REPO_CLONE_SPACE
+          
+      - name: Verify example.properties exists
+        run: |
+          set -ex
+          if [ -f ASHE/src/main/resources/example.properties ]; then 
+            echo "example.properties found"
+          else 
+            echo "example.properties not found"
+            exit 1
+          fi
+
+      - name: Rename example.properties to config.properties file
+        run: |
+          set -ex
+          mv ASHE/src/main/resources/example.properties ASHE/src/main/resources/config.properties
+          if [ -f ASHE/src/main/resources/config.properties ]; then 
+            echo "config.properties created"
+          else 
+            echo "config.properties not created"
+            exit 1
+          fi
+
+      - name: Give write permissions to config.properties
+        run: |
+          set -ex
+          chmod +w ASHE/src/main/resources/config.properties
+          ls -l ASHE/src/main/resources/config.properties
+
+      - name: Update ASHE Config File to update SPECIMIN path
```

Comment:
"Specimin" is a name, not an acronym. So, it should not be styled as "SPECIMIN", since it doesn't stand for anything.